src/aleph_vrf/coordinator/vrf.py

```diff
@@ -66,12 +62,26 @@ async def post_executor_api_request(url: str, model: Type[M]) -> M:
             return model.parse_obj(response["data"])
 
 
+async def prepare_executor_api_request(url: str) -> bool:
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url, timeout=120) as resp:
+            if resp.status != 200:
+                raise ExecutorHttpError(
+                    url=url, status_code=resp.status, response_text=await resp.text()
+                )
```

Comment:
Why not rely on the standard `raise_for_status()` ? We use it in all the other projects.
Even if the goal is to raise a specific error, this would provide additional context when analyzing the stack trace.

```suggestion
            try:
                resp.raise_for_status()
            except aiohttp.ClientResponseError as error:
                raise ExecutorHttpError(
                    url=url, status_code=resp.status, response_text=await resp.text()
                ) from error
```

Reply:
This s already fixed in the other PR:

https://github.com/aleph-im/aleph-vrf/pull/26/files#diff-1d589daebee84d1efd29eb44cc810506c62a927c3decc9d8c2196d33eb9f2c48R65-R77